package.json

```diff
@@ -66,7 +66,7 @@
     "cross-env": "^7.0.3",
     "eslint": "^8.51.0",
     "eslint-config-digitalbazaar": "^3.0.0",
-    "eslint-plugin-jsdoc": "^39.3.2",
+    "eslint-plugin-jsdoc": "^49.0.0",
```

Comment:
Ought it be fixed?

```suggestion
  "eslint-plugin-jsdoc": "49.0.0",
```

Reply:
So all the dependencies? They are not, so I just left it